hstream/src/HStream/Server/Exception.hs

```diff
@@ -86,6 +86,15 @@ mkExceptionHandle retFun cleanFun = flip catches [
 defaultExceptionHandle :: IO (ServerResponse 'Normal a) -> IO (ServerResponse 'Normal a)
 defaultExceptionHandle = mkExceptionHandle returnErrResp $ return ()
 
+appendStreamExceptionHandle :: IO (ServerResponse 'Normal a) -> IO (ServerResponse 'Normal a)
+appendStreamExceptionHandle action = defaultExceptionHandle $ catches action [
+  Handler (\(err :: Store.NOTFOUND) -> do
```

Comment:
is `Store.NOTFOUND` too general? What if this logid really doesn't exist?

Reply:
as the comment https://github.com/hstreamdb/hstream/blob/3709d280cbb372994c5c4d8fbb299ccfe41abfb6/hstream-store/HStream/Store/Internal/LogDevice/LogConfigTypes.hs#L433 NOTFOUND is probably thrown, should we ignore it?